students/Jaidjen/lesson03/Mailroom1.py

```diff
@@ -1,53 +1,60 @@
-import sys  # imports go at the top of the file
+import sys 
 
 
-Mail  =[('James Smith',[2000.25, 500, 27.50]),
+donors  =[('James Smith',[2000.25, 500, 27.50]),
         ('Dorothy Parker',[3175, 475.45, 92.30]),
         ('Dohgyu Hwang',[10670, 350.19,175]),
         ('Sally Yu', [100, 925]),
         ('Jenny Ramos',[25.14])]
 
 prompt = "\n".join(("Functions:",
           "Please choose from below options:",
-          "1 - Send a Thank You email",
+          "1 - Send a Thank You to donors",
           "2 - Create a Report",
           "3 - Quit",
           ">>> "))
 
 
-def view_mail(Mail):
+def add_donors():
+    print("If you want a list of donors please type 'list' ")
+    new_donors = input("Please enter the full name of the donor or select list?  ")
+    for i in range(len(donors)):
+        while True:
+            if new_donors == "list":
+                print("{:<20s}".format(donors[i][0]))
+                break
+            else:
+                donation_amt = int(input("Please enter the donation amount: "))
+                for donor in donors:
+                    if donor[0] == new_donors:
+                        donors.append()
+                        break
+                    else:
```

Comment:
also, for and else are on the same indentation. `else` should be part of your for loop not if statement.